### PR TITLE
localization(i18n)

### PR DESCRIPTION
**Pull request by tuanngoc1601**
Localization

### CODE REVIEW

**Comment by tuanngoc1601** [comment]
ready

File: lang/en.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    "Too Many Requests": "Too Many Requests",
+    "Unauthorized": "Unauthorized",
+    "Update Password": "Update Password",
+    "Update your account's profile information and email address.": "Update your account's profile information and email address.",
+    "Verify Email Address": "Verify Email Address",
+    "Whoops!": "Whoops!",
+    "You are receiving this email because we received a password reset request for your account.": "You are receiving this email because we received a password reset request for your account.",
+    "You're logged in!": "You're logged in!",
+    "Your email address is unverified.": "Your email address is unverified."
+}
```
Comment (ghost) [inline lang/en.json]: missing empty line eof

File: lang/en/http-statuses.php [hunk 2]
```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    '0'            => 'Unknown Error',
```
Comment (ghost) [inline lang/en/http-statuses.php]:
```suggestion
    '0' => 'Unknown Error',
```

File: lang/vi.json [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    "Too Many Requests": "Quá Nhiều Yêu Cầu",
+    "Unauthorized": "Không Được Phép",
+    "Update Password": "Cập nhật mật khẩu",
+    "Update your account's profile information and email address.": "Cập nhật thông tin hồ sơ tài khoản và địa chỉ email của bạn.",
+    "Verify Email Address": "Xác Minh Địa Chỉ Email",
+    "Whoops!": "Rất tiếc!",
+    "You are receiving this email because we received a password reset request for your account.": "Bạn nhận được email này vì chúng tôi đã nhận được yêu cầu đặt lại mật khẩu cho tài khoản của bạn.",
+    "You're logged in!": "Bạn đã đăng nhập!",
+    "Your email address is unverified.": "Địa chỉ email của bạn chưa được xác minh."
+}
```
Comment (ghost) [inline lang/vi.json]: missing empty line eof

**Comment by tuanngoc1601** [comment]
em sửa rồi ạ

**Comment by tuanngoc1601** [comment]
ready

File: app/Http/Middleware/Localization.php [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+class Localization
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Closure(\Illuminate\Http\Request): (\Symfony\Component\HttpFoundation\Response)  $next
+     */
+    public function handle(Request $request, Closure $next): Response
+    {
+        if(Session::get('lang')) {
```
Comment (ghost) [inline app/Http/Middleware/Localization.php]:
```suggestion
        if (Session::get('lang')) {
```

File: app/Http/Middleware/Localization.php [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+            App::setLocale(Session::get('lang'));
+        }
+        return $next($request);
```
Comment (ghost) [inline app/Http/Middleware/Localization.php]:
```suggestion
        }

        return $next($request);
```

File: lang/en.json (hunk 1 above)
Comment (ghost) [reply lang/en.json]: not fixed yet

File: lang/en/validation.php [hunk 6]
```diff
@@ -0,0 +1,217 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    'accepted'             => 'The :attribute must be accepted.',
```
Comment (ghost) [inline lang/en/validation.php]:
```suggestion
    'accepted' => 'The :attribute must be accepted.',
```

File: lang/vi.json (hunk 3 above)
Comment (ghost) [reply lang/vi.json]: not fixed

File: lang/vi/http-statuses.php [hunk 7]
```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    '0'            => 'Lỗi Không Xác Định',
```
Comment (ghost) [inline lang/vi/http-statuses.php]:
```suggestion
    '0' => 'Lỗi Không Xác Định',
```

**Comment by tuanngoc1601** [comment]
ready

**Comment by ghost** [comment]
chưa fix hết comment.

**Comment by ghost** [comment]
commented

**Comment by tuanngoc1601** [comment]
ready

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
merged

**Pull request merged**